src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -94,6 +95,7 @@
       .put("INVALID_DYNAMIC_LINK_DOMAIN", "invalid-dynamic-link-domain")
       .build();
 
+  static final int MAX_LIST_PROVIDER_CONFIGS_RESULTS = 100;
```

Comment:
Is 100 the right value to use as a maximum? The Go implementation uses 100 but I don't think the Node implementation has a limit at all. It's not clear what limit the server enforces.